src/components/pages/blog-post/blog-post.tsx

```diff
@@ -1,19 +1,25 @@
-import React from 'react';
-import { Layout } from '../../layout/layout';
-import { LeadboxProps } from '../../content/leadbox/leadbox';
+import { Options } from '@contentful/rich-text-react-renderer';
+import { BLOCKS, INLINES, MARKS } from '@contentful/rich-text-types';
+import { GatsbyImage } from 'gatsby-plugin-image';
 import { useTranslation } from 'gatsby-plugin-react-i18next';
-import { MarkdownAst } from '../../legacy/markdown/markdown-ast';
-import SharePanel from './share-panel';
-import FollowPanel from './follow-panel';
+import { renderRichText } from 'gatsby-source-contentful/rich-text';
+import React from 'react';
 import styled from 'styled-components';
-import { up } from '../../support/breakpoint';
-import { BlogPostMarkdown, BreadcrumbEntry } from '../../../types';
-import { BlogHero } from '../../content/heroes/blog-hero';
+import { BlogArticleQueryData } from '../../../templates/blog-post';
+import { BreadcrumbEntry } from '../../../types';
 import { BlogHeader } from '../../content/blog-header/blog-header';
+import { BlogHero } from '../../content/heroes/blog-hero';
+import { LeadboxProps } from '../../content/leadbox/leadbox';
 import { LONG_DATE_FORMAT, useLocaleFormat } from '../../i18n-helpers';
+import { Layout } from '../../layout/layout';
+import cSC from '../../legacy/markdown/custom-components';
```

Comment:
Very hard to read, maybe use a longer name?

Reply:
Renaming to `customComponents` 🎉